sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/TestJsonData.scala

```diff
@@ -229,6 +229,11 @@ private[json] trait TestJsonData {
       """{"date": "27/10/2014 18:30"}""" ::
       """{"date": "28/01/2016 20:00"}""" :: Nil))(Encoders.STRING)
 
+  def badRecords: Dataset[String] =
```

Comment:
if it's only used in one test, let's move it to that test

Reply:
moved